rviz_common/CMakeLists.txt

```diff
@@ -31,7 +31,17 @@ find_package(ament_cmake REQUIRED)
 # do find_package(rviz_ogre_vendor) first to make sure the custom OGRE is found
 find_package(rviz_ogre_vendor REQUIRED)
 
-find_package(Qt5 REQUIRED COMPONENTS Widgets)
+if(MSVC)
```

Comment:
instead of using in the condition windows why don't use the Qt version instead?

Reply:
addressed here https://github.com/ros2/rviz/pull/1187/commits/397709398eb7396637ccb9a7bbdfd49234fd3685